### DISCUSSION TITLE
10.20.21 Webinar Transcript (?)

### DISCUSSION THREAD

**Discussion by Data-san** [CMSgov/price-transparency-guide#228] [Q&A] [answered]
Apologies if I missed it. Believe I heard there would be a transcript of today's webinar. If so, where will it be posted? Thanks much.

**Comment by shaselton-usds** [comment, marked as answer]
You should be able to view it here: https://cms.gov/healthplan-price-transparency/resources/webinars

Let me know if you have any issues.

**Reply by Data-san** [reply]
Thank you!

**Reply by kavkuk** [reply]
@shaselton-usds I do not see the transcript at the link you provided.  Will it be posted at a later time?  I just see a place to register for the Webinar on the 10/20 as well as a link to post questions, but no transcript.  Am I missing something?

**Reply by shaselton-usds** [reply]
@kavkuk There should be a link to download a word document that contains the script of the webinar now. Let me know if you run into any trouble.

**Reply by kavkuk** [reply]
@shaselton-usds I see it now. Thank you!
